### ISSUE TITLE
Installing with lxml-5.2.1 ImportError: cannot import name '_ElementStringResult' from 'lxml.etree'

### ISSUE THREAD

**Issue by svoss** [scrapinghub/extruct#232]
It seems that installing extruct via `pip install extruct` now automatically install lxml 5.2.11 which causes the following import error:

```
ImportError: cannot import name '_ElementStringResult' from 'lxml.etree'
```

I am running python 3.10, have been able to fix it by installing lxml[html_clean]==4.9.3.

Seems to be fixed in the master branch, maybe make a new version?

**Comment by michael-genson** [comment]
This is fixed, just not in a release yet: https://github.com/scrapinghub/extruct/pull/217

**Comment by tonal** [comment]
> This is fixed, just not in a release yet: #217

When can I wait for the release?

**Comment by wRAR** [comment]
We've just released 0.17.0.
